bin/exporter

```diff
@@ -23,9 +24,12 @@ client    = ConnectionPool::Wrapper.new(size: 15, timeout: 3) do
                           init_command: "SET @@SESSION.MAX_EXECUTION_TIME=31536000")
             end
 
+s3_client = Aws::S3::Client.new(region: region)
+
 AuroraBootstrapper::Exporter.new( client: client,
                                   prefix: prefix,
                            export_bucket: bukkit,
                       blacklisted_tables: blacklisted_tables,
                       whitelisted_tables: whitelisted_tables,
-                      blacklisted_fields: blacklisted_fields ).export!
\ No newline at end of file
+                      blacklisted_fields: blacklisted_fields,
+                      s3_client: s3_client ).export!
```

Comment:
if the client never changes, what's the benefit of passing it in here? Can it be created on demand?

Reply:
Sure.